tests/unittests/AddressBook/MultiValueTests.mm

```diff
@@ -0,0 +1,233 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#include <TestFramework.h>
+
+#import <CoreFoundation/CoreFoundation.h>
+#import <Foundation/Foundation.h>
+
+#import "UWP/WindowsApplicationModelContacts.h"
+#import <AddressBook/ABAddressBook.h>
+#import <AddressBook/ABRecord.h>
+#import <AddressBook/ABPerson.h>
+#import <AddressBook/ABMultiValue.h>
+#import "ABContactInternal.h"
+
+class AddressBookMultiValueQueryTest
+    : public ::testing::TestWithParam<::testing::tuple<ABPropertyID, CFStringRef, CFIndex, NSString*, CFIndex>> {
+protected:
+    virtual void SetUp() {
+        WACContact* contact = [WACContact make];
+
+        addPhone(contact, WACContactPhoneKindHome, @"0000000000");
```

Comment:
The reference platform accepts any arbitrary String

---
In reply to: [75181698](https://github.com/Microsoft/WinObjC/pull/884#discussion_r75181698) [](ancestors = 75181698)